addons/binding/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/convention/homeassistant/CFactory.java

```diff
@@ -35,18 +36,19 @@
     /**
      * Create a HA MQTT component. The configuration JSon string is required.
      *
-     * @param thingUID The Thing UID that this component will belong to.
-     * @param haID The location of this component. The HomeAssistant ID contains the object-id, node-id and
-     *            component-id.
-     * @param configJSON Most components expect a "name", a "state_topic" and "command_topic" like with
-     *            "{name:'Name',state_topic:'homeassistant/switch/0/object/state',command_topic:'homeassistant/switch/0/object/set'".
+     * @param thingUID       The Thing UID that this component will belong to.
+     * @param haID           The location of this component. The HomeAssistant ID contains the object-id, node-id and
+     *                           component-id.
```

Comment:
Not a problem for me, but for others probably. Go to eclipse window->preferences->Java->code style->formatter and make sure that "ESH" is the active profile. Click apply, even if it is. Then go back to this file and let it be reformatted again.